.circleci/config.yml

```diff
@@ -289,9 +289,9 @@ workflows:
   build-check:
     jobs:
       - prefetch-crates
-      - nightly-build-check:
-          requires:
-            - prefetch-crates
+      # - nightly-build-check:
```

Comment:
Any reason to leave these commented out, or should we remove it entirely? Will we potentially turn this back on?

Reply:
I think so, I just don't have time to look into it right now, and it might be something that'll get sorted out eventually. Maybe we could have someone experienced in CI to look into it. I know we have a few on staff. @GregTheGreek ?

Reply:
Well it passes in the meantime, so I guess we can go ahead and stick it through.

Reply:
It might also be nice to fix MacOS builds too, you know?

Reply:
True, maybe they can be lumped up into a single issue?

Reply:
The underlying problem is that a dependency is using features which are deprecated in nightly. @ec2 tried to phase out the offending crate, but it turned out to be a longer task than he expected